eng/templates/official/jobs/ci-e2e-tests.yml

```diff
@@ -0,0 +1,161 @@
+# This workflow will install Python dependencies and run end to end tests with single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI E2E tests
```

Comment:
entire file - this format/syntax will probably not work with ADO, it would need conversion from Github action to ADO pipeline

Reply:
Updated